content/faq/shapeshift.md

```diff
@@ -0,0 +1,44 @@
+---
+title: How can I convert my crypto into LBC?
+category: wallet
+---
+
+The ability to convert your cryptoassets into LBRY Credits (LBC) is available directly within the LBRY app, powered by [ShapeShift](https://info.shapeshift.io/about). LBRY Inc. does not handle any of the incoming funds and the LBC are sent from ShapeShift directly. 
+
+*Note: ShapeShift is unavailable in New York and Washington (and possibly surrounding areas if your ISP's goelocation is not accurate). Please see [ShapeShift](https://shapeshift.io) for more information.* 	 
+
+## Convert Crypto to LBC
+1. Open the LBRY app, access the Wallet (bank icon next to the Publish button) and click on **Get Credits**
+
+2. In the **Convert Crypto to LBC** section, choose from BTC, BCH, DASH, ETH, LTC or XMR to convert into LBC
+<img src="https://spee.ch/3/convertcrypto1.JPG" width="80%" height="80%">
+
+3. Review the given rate of exchange and min/max amount. ShapeShift charges a small [fee](https://info.shapeshift.io/about) for the transaction
+4. Enter the return address for the cryptoasset in case something were to go wrong with the process
+<img src="https://spee.ch/7/convertcrypto2.JPG" width="80%" height="80%">
+
+5. Click **Begin Conversion**  to start your request. You will now be presented with the deposit address for your conversion.
+<img src="https://spee.ch/c/convertcrypto3.JPG" width="80%" height="80%">
+
+6. Using your crypto wallet, send any amount between the min and max to the deposit address specified. Click **VIEW THE STATUS ON SHAPESHIFT.IO** to track in real-time. You can also bookmark this transaction for your records. We are not currently storing any information about the transaction after it confirms and you receive the LBC in your wallet.
+<img src="https://spee.ch/4/convertcrypto4.JPG" width="80%" height="80%">
+
+7. Once your transaction is confirmed, you will be presented with the completion screen. Click **Done** to start a new conversion. 
+<img src="https://spee.ch/2/convertcrypto5.JPG" width="80%" height="80%">
+
+8. Verify that LBC has been received on the **History** tab.
+<img src="https://spee.ch/8/convertcrypto6.JPG" width="80%" height="80%">
+
+9. Thanks for acquiring some LBC! 
+
+### I need help with my conversion, who can I reach out to?
+
+If you initiated your conversion but are having trouble sending the cryptoasset or receiving LBC, please reach out to [ShapeShift directly](https://shapeshift.zendesk.com/hc/en-us/requests/new) as LBRY is not responsible for the conversion process. 
+
+### How else can I acquire LBRY Credits?
+
+Please see our [earn credits](https://lbry.io/faq/earn-credits) FAQ on how to acquire additional LBC.
+
+### I need help with the LBRY app, who can I reach out to?
```

Comment:
Should this be in this document? I'm open to the idea of something like this as a "catch-all" footer on all FAQ articles.

Reply:
I like it there and it's already included in a bunch of the other FAQs revolving around the app. Will continue to standardize on including it.